src/styles/blocks/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You can use just - https://developer.mozilla.org/ru/docs/Web/CSS/background
Instead of `background-repeat`, `background-position` and `background-image`